pkgs/development/python-modules/jupyter-server-ydoc/default.nix

```diff
@@ -13,6 +14,18 @@
   jupyter-collaboration,
 }:
 
+let
+  # pycrdt-websocket got breaking changes in 0.16.0 which break this package.
+  pycrdt-websocket' = pycrdt-websocket.overridePythonAttrs (rec {
```

Comment:
given that pycrdt-websocket only has one user, should we just not upgrade it ? otherwise we introduce version collision ?

Reply:
Yes makes sense! I'll mark this one as a draft.

Reply:
Such overrides wouldn't be allowed within `python3Packages` anyway.